### PR TITLE
at/cb-147-create-db-mock-for-payment-capacity

### PR DESCRIPTION
**Pull request by Angelbt12**
Create DB mock for "payment capacity"
image

### CODE REVIEW

**Review by Andresbl123 — APPROVED** [review]
cumple requerimiento

File: src/mocks/creditProfileInfo/paymentCapacity.mock.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+export const payment_capacityInfo = [
+  {
+    credit_request_id: "999990",
+    payment_capacity: {
+      available_value: 955500,
+      available_percentage: 20,
+      base_income: 3000000,
+      percentage_used: 60,
```
Comment (carin91) [inline src/mocks/creditProfileInfo/paymentCapacity.mock.ts:8]: This is not necessary, should be calculated based con the available_percentage (used_percentage = 1 - available_percentage).

File: src/mocks/creditProfileInfo/paymentCapacity.mock.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+export const payment_capacityInfo = [
```
Comment (carin91) [inline src/mocks/creditProfileInfo/paymentCapacity.mock.ts:1]: The task considers rendering the information on every fieldset based on the mock.
image
As an example, you could review the code which controls the risk scoring rendering
image

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/mocks/creditProfileInfo/paymentCapacity.mock.ts (hunk 1 above)
Comment (Angelbt12) [reply src/mocks/creditProfileInfo/paymentCapacity.mock.ts]: ok

File: src/mocks/creditProfileInfo/paymentCapacity.mock.ts (hunk 2 above)
Comment (carin91) [reply src/mocks/creditProfileInfo/paymentCapacity.mock.ts]: This activity is still pending

**Review by carin91 — APPROVED** [review]

**Pull request merged**